Client/Entitlements/FennecApplication.entitlements

```diff
@@ -4,6 +4,8 @@
 <dict>
 	<key>aps-environment</key>
 	<string>development</string>
+	<key>com.apple.developer.authentication-services.autofill-credential-provider</key>
+	<true/>
```

Comment:
I pushed a change to add this entitlement to  `FirefoxApplication.entitlements` and `FirefoxBetaApplication.entitlements`.

(Ignoring `FennecEnterprise` and `FirefoxEnterprise` because these are not used anymore.)